src/styles/utils/reset.css

```diff
@@ -0,0 +1,41 @@
+html,
+body,
+div,
+span,
+h1,
+h2,
+p,
+a,
+em,
+img,
+ul,
+li,
+footer,
+header {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  vertical-align: baseline;
+}
```

Comment:
most of this elements don't have these styles, for example `div` doesn't have default `margin, padding, border` so you don't need reset it